src/AppKit/NSOpenGLPixelFormat.cs

```diff
@@ -58,7 +58,11 @@ public NSOpenGLPixelFormat (NSOpenGLPixelFormatAttribute[] attribs) : base (NSOb
 		{
 			var list = new List<NSOpenGLPixelFormatAttribute> ();
 			for (int i = 0; i < args.Length; i++){
-				var v = (NSOpenGLPixelFormatAttribute) args [i];
+				NSOpenGLPixelFormatAttribute v;
+				if (args[i] is NSOpenGLPixelFormatAttribute)
+					v = (NSOpenGLPixelFormatAttribute) args [i];
+				else
+					v = ((NSOpenGLPixelFormatAttribute) (int)args [i]);
```

Comment:
So after more research, what's actually happening here is that NSOpenGLPixelFormatAttribute.OpenGLProfile in the user's code requires a value, which uses the NSOpenGLProfile enum (the Version3_2Core in this case).  So the solution for that issue is just adding a case to the switch for NSOpenGLPixelFormatAttribute.OpenGLProfile that shares the code that grabs the next item in the array as the value.  I've almost got a new commit ready to go that deals with this and a couple more changes (still need to write tests as well)